src/objects/application.ts

```diff
@@ -1,16 +1,32 @@
 import { Application } from 'pixi.js'
-import { stage } from '@/objects/stage'
+import { emitter } from '@/objects/events'
 
-export let application: Application
+export let application: Application | null = null
 
-export function createApplication(): void {
+export function startApplication(): void {
+  emitter.on('stageCreated', createApplication)
+}
+
+export function stopApplication(): void {
```

Comment:
Do we not need to detach the emitter here?

Reply:
Nvm I see we do that in the parent caller - but I see this method being called explicitly and not the `stop` method

Reply:
`stopApplication` gets called in `stop`. Its my intent that the start* and stop* methods should never get called individually but only by the root start and stop methods. 

Though I could see an argument for having every `emitter.on` in a start method have a corresponding `emitter.off` in the stop method.

I think its probably still a good idea to just clear all the event listeners when the graph is stopped though.

Reply:
Yea probably both are needed - worth noting for now, not worth slowing down any work